tests/baselines/reference/user/uglify-js.log

```diff
@@ -128,12 +139,15 @@ node_modules/uglify-js/lib/propmangle.js(70,45): error TS2339: Property 'prototy
 node_modules/uglify-js/lib/propmangle.js(80,29): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/propmangle.js(94,30): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/propmangle.js(146,29): error TS2554: Expected 0 arguments, but got 1.
+node_modules/uglify-js/lib/propmangle.js(163,26): error TS2554: Expected 2 arguments, but got 1.
```

Comment:
TreeTransformer takes two functions and the second is indeed optional, but it's not immediately obvious because there's no naming convention and the optionality check is hundreds of lines from the parameter.